ui/app/controllers/jobs/job/task-group.js

```diff
@@ -66,9 +66,10 @@ export default class TaskGroupController extends Controller.extend(
   })
   shouldShowScaleEventTimeline;
 
-  @computed('model.job.runningDeployment')
+  @computed('model.job.runningDeployment', 'model.job.namespace')
```

Comment:
@DingoEatingFuzz I'm getting an error `use brace expansion` but if write it like:

`('model.{job.namespace, job.runningDeployment}')`

OR

`('model.{namespace, runningDeployment}')`

I get `Use of undeclared dependencies in computed property`. Is that because we're going based on nested keys?

Reply:
you want a third option:

```js
'model.job.{runningDeployment,namespace}'
```

Also, it at least used to be the case that spaces after commas in dependent keys would not work.